### PR TITLE
[vercel deploy] Note: re: 404 page issue

### PR DESCRIPTION
**Pull request by sarah11918**
- New or updated content

Moved content added to the SSR integration README in this PR https://github.com/withastro/astro/pull/4299 which attempted to document a workaround to this Vercel bug: https://github.com/withastro/astro/issues/4164 but did so in the wrong place and led to confusion.

Removed the content from the SSR adapter integration in this PR: https://github.com/withastro/astro/pull/4551

Closes https://github.com/withastro/docs/pull/1412

Totally open to edits! (Too Vercel blamey?) Just trying to quickly capture the info here.

### CODE REVIEW

**Comment by delucis** [comment]
I’m going to run some tests on this one. This workaround has some notable side effects by disabling `trailingSlash` so want to double check this is definitely required.

File: src/pages/en/guides/deploy/vercel.md [hunk 1]
```diff
@@ -17,6 +17,17 @@ Your Astro project can be deployed to Vercel as a static site, or as a server-si
 
 Your Astro project is a static site by default. You don’t need any extra configuration to deploy a static Astro site to Vercel. 
 
+:::note
+There is currently a Vercel issue displaying a 404 page on Astro websites. Until this is fixed, you can add the following config file at the root of your project:
+
+```json title="vercel.json"
+{
+  "cleanUrls": true,
+  "trailingSlash": false
```
Comment (delucis) [inline src/pages/en/guides/deploy/vercel.md:26]: In my testing, just setting `"cleanUrls": true` was enough to get the 404 page working, so I think we can narrow this down to that:

```suggestion
```

This is also less of a serious side effect than the `trailingSlash` option, which is more likely to conflict with desired behaviour. It’s still _definitely_ a bug on Vercel’s end that this is necessary, so curious to try to keep tabs on this so we know when this is fixed.

**Comment by sarah11918** [comment]
@delucis With the code update, good to go?

**Review by delucis — APPROVED** [review]
LGTM!

**Pull request merged**